phpunit/fonts-library/class-wp-fonts-library-test.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Tests for Fonts Library class
+ *
+ * @package Gutenberg
+ */
+
+class WP_Fonts_Library_Test extends WP_UnitTestCase {
+
+	function test_get_fonts_directory() {
+		$this->assertTrue(
+			str_ends_with( WP_Fonts_Library::get_fonts_directory(), '/wp-content/fonts' )
```

Comment:
This can be problematic perhaps. Seen quite a few cases in the past where the server is set to allow writes from PHP only to `wp-content/uploads`. Any particular reason why `wp-content/uploads/fonts` should not be used for fonts? (Assuming this is to store fonts uploaded by the users.)